### DISCUSSION TITLE
Plotly Plot Not Resizing To Take Up Full Page

### DISCUSSION THREAD

**Discussion by Jamesong7822** [datapane/datapane#378] [Help]
### Summary

Report Baselink is here: [https://cloud.datapane.com/reports/dA6xO47/387295849-0/](https://cloud.datapane.com/reports/dA6xO47/387295849-0/)
I have labelled the plot block as "Plotly" and attempted to access it via the following url:[https://cloud.datapane.com/reports/dA6xO47/387295849-0/embed/?block_name=Plotly](https://cloud.datapane.com/reports/dA6xO47/387295849-0/embed/?block_name=Plotly)

However, the plot does not automatically resize to the full window and still has whitespace margins which is not something i want.
![image](https://github.com/datapane/datapane/assets/42574761/17eef57b-036a-4588-8194-6c7239c584e8)
On top of that, there's weird text on the top about `:root { --dp-accent-color: #4E46E5; --dp-bg-color: #FFF; --dp-text-align: left; --dp-font-family: Inter, ui-sans-serif, system-ui; }` which I have no idea how to get rid off.

I created the report via:
```
plotUrl = dp.upload_report(dp.Plot(fig, name="Plotly"), name=name, publicly_visible=True, 
                               overwrite=True, open=False, 
                               formatting=dp.Formatting(width=dp.Width.FULL)).web_url
```

**Comment by mands** [comment]
Hi there, thanks for raising this - we'll take a look and try to fix it asap.
